doc/md/fundamentals/2-actors/5-messaging.md

```diff
@@ -50,7 +50,7 @@ In this example, a publisher canister maintains a list of subscribers and sends
 
 ```motoko no-repl
 import Subscriber "canister:subscriber";
-import Array "mo:base/Array";
+import Array "mo:core/Array";
 
 actor Publisher {
     stable var subscribers : [Subscriber.Subscriber] = [];
```

Comment:
Making a note that we may want to revisit these previously incomplete pub/sub examples.